utils/models.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+
+from django.db import models
+
+
+class BaseModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def save(
+        self, force_insert: bool = False, force_update: bool = False, using: Any = None, update_fields: Any = None
+    ) -> None:
+        self.full_clean()
+        super().save()
```

Comment:
You need to pass all the `save` parameters because right now if anything use them you just don't allow them to use them correctly